database/migrations/default/1744826895218_ems_remove_match_status_constraint/up.sql

```diff
@@ -0,0 +1,47 @@
+alter table ems__incidents
+drop constraint matched_crash_pks_non_null,
+add column created_by text not null default 'system',
+add column updated_by text not null default 'system';
+
+comment on column ems__incidents.matched_crash_pks is 'The IDs of multiple crashes that were found to match this record. Set via trigger.''';
+
+--
+-- Trigger function that sets the ems__incident `crash_pk` based on the
+-- provided person_id and updates the crash_pk of all EMS records that
+-- share the same incident number
+--
+create or replace function update_person_crash_ems_match()
```

Comment:
This trigger covers two use cases. 

First, it allows the editor to assign a person ID to an EMS record, and this trigger takes care of ensuring that the `crash_pk` column is correct. Otherwise, we have a situation where the `crash_pk` column is out of sync with the `person_id` column. 

Consider this example:
- The crash-EMS trigger automatically assigns a crash PK to an EMS record
- A user discovers that this match was incorrect, and uses the UI to assign a person ID from a *different crash*
- This database trigger detects the the EMS record `person_id` has changed, and updates the record's crash_pk to match that of the person ID